### PR TITLE
Vimeo video fullscreen

### PR DESCRIPTION
**Pull request by neu5**

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -89,9 +91,51 @@ $().ready(() => {
...
+        marginTop: s.marginTop,
+        marginLeft: s.marginLeft
+    })
+  }
+
+  if (IS_HOMEPAGE) {
+    scrollSpy();
+    resizeVideo();
+
+    $(window).on('resize', () => {
```
Comment (CezaryDanielNowak) [inline src/js/main.js:133]: are you planning say "f.off", IE and Safari users? Or it's transpiled by babel first?

File: src/sass/_common.scss [hunk 2]
```diff
@@ -16,3 +16,14 @@ ul {
...
 }
+
+.embed-container {
+  position: relative;
+  overflow: hidden;
+  width: 100%;
+  height: 100%
+}
+
+.embed-container iframe, .embed-container object, .embed-container embed {
```
Comment (CezaryDanielNowak) [inline src/sass/_common.scss]:
```
.embed-container {
  iframe, object, embed {
    ...
  }
}
```

File: src/js/main.js [hunk 3]
```diff
@@ -89,9 +91,51 @@ $().ready(() => {
     setActive($navLinks, $navLinks[$sectionsObject.indexOf(closestSection)]);
   }
 
-  scrollSpy();
+  let container = $('#embed_container');
+  let video = $('#video');
+
+  function resizeVideo() {
```
Comment (CezaryDanielNowak) [inline src/js/main.js:97]: why not width: `100vw; height: 100vh;` ?

File: views/pages/homepage.ejs [hunk 4]
```diff
@@ -128,10 +128,14 @@
     </ul>
   </nav>
   <main id="home" class="section intro">
-    <iframe src="https://player.vimeo.com/video/155022290?autoplay=1&loop=1" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+    <div id="embed_container" class='embed-container'>
```
Comment (CezaryDanielNowak) [inline views/pages/homepage.ejs]: try not to mix single and double quotes

File: src/js/main.js (hunk 1 above)
Comment (neu5) [inline src/js/main.js:133]: it's tarnspiled by babel

File: src/sass/_common.scss (hunk 2 above)
Comment (neu5) [inline src/sass/_common.scss]: that was copy paste, good catch

File: views/pages/homepage.ejs (hunk 4 above)
Comment (neu5) [inline views/pages/homepage.ejs]: I really missed that :smile:  thanks :kissing:

File: src/js/main.js (hunk 3 above)
Comment (neu5) [inline src/js/main.js:97]: because it won't work, we need to get rid of black margin in the video

**Pull request merged**